src/triage/experiments/validate.py

```diff
@@ -328,16 +328,19 @@ def _validate_imputations(self, aggregation_config):
         agg_types = ["aggregates", "categoricals", "array_categoricals"]
 
         for agg_type in agg_types:
+            logging.info('agg_type:{}'.format(agg_type))
```

Comment:
Use logging variables instead of formatted strings:
`logging.info('agg_type: %s', agg_type)`

This has two advantanges:
- logging module doesn't do the interpolation unless the message is actually visible (minor win)
- and enables log aggregators to more intelligently aggregate logfiles. For instance, if a message gets logged multiple times with different arguments, log aggregators are smart enough to aggregate them together but this is not possible with formatted strings.

And the intros should be a bit more descriptive. 'Checking imputation rules for aggregation type %s', 'Checking imputation rules for aggregation %s', 'Checking imputation rules for metric %s'